### DISCUSSION TITLE
About using spa site on hash routes

### DISCUSSION THREAD

**Discussion by cnguu** [umami-software/umami#1142] [Q&A]
I added a spa site that uses hash routing, but only the `/` page has views statistics.

> Hash route like: `www.domain.com/#/a/b`

I'm not sure if I've configured it wrong or if the umami doesn't support this.

**Comment by NeoN0x** [comment]
``Hello,

I'm facing the same issue.

For now, as a workaround, I copypasted unami.js to include it directly, and replaced the route to replace it in an inline way 

```js
// From
website: (sth)
// To
website: (mywebsite uid)

// From (3 times)
url: D
// To
url: D.replace('#','')

// From
referrer: e
// To
referrer: e.replace('#', '')
```
